### DISCUSSION TITLE
Request for Retention or Redeployment of V5 Documentation and Demos

### DISCUSSION THREAD

**Discussion by JoyNop** [fabricjs/fabric.js#10328] [Q&A] [answered]
I found that the official documents have been replaced with new V6 documents, but there are still relatively few demos. Can the fabric administrator retain or redeploy the V5 documents and demos for our reference? Our existing software needs the support of V5 documents and DEMOs. Thank you very much.

**Comment by wangyong991** [comment]
I have collected some Chinese information, I hope it will be helpful to you
https://github.com/Rookie-Birds/Fabric-Tutorial_zh-CN/tree/master
https://gitee.com/k21vin/fabricjs-demo

**Reply by JoyNop** [reply]
非常感谢~这些文档很不错 ，[k21vin/fabricjs-demo](https://gitee.com/k21vin/fabricjs-demo)很早之前就搜藏啦😁

**Comment by asturur** [comment]
Is not easy to keep 2 website up and the github will allow me to have just one per org.
I would need to create a new org move the repo there and publish on a new domain

**Reply by Smrtnyk** [reply]
have you considered having versioned docs @asturur  ?
maybe by using docusaurus or vitepress
https://docusaurus.io/
https://vitepress.dev/

**Reply by asturur** [reply]
I m not restarting another stack, it took me a while to do what we have and i tried docosaurus but i ended up discaring it because i couldn't easily build a custom theme for it.

I do not think people care for the api docs but they want the pages and the demos. If it is just about typedoc i m sure something can be done easily.

In the folder old docs anyway there are most of the old documentation in the new website.

**Reply by JoyNop** [reply]
When working with Fabric.js to create a new canvas, I often refer to the demo documentation for inspiration. It can sometimes be even more helpful than the API docs, though both are equally important. Since my current project uses v5 and it’s not practical to upgrade right now, I set up a v5 documentation site using the existing code repository for everyone to reference. 

（However, I’m not sure how long it will remain accessible.🤔🤔🤔）
https://v5.fabric.doc.ieasn.com/

**Comment by asturur** [comment, marked as answer]
Deployed https://fabric5.fabricjs.com hopefully everything works as it should.

**Reply by asturur** [reply]
[JoyNop](https://github.com/JoyNop) if you can mark this as answer you can help me. thanks.

**Reply by aswind7** [reply]
![image](https://github.com/user-attachments/assets/a46fd1f2-990b-4754-8745-5808cedbc705)


@asturur It would be great if you could provide documentation instructions and github link address for v5 in the header of https://fabricjs.com/, which would make things much easier for people.

**Reply by asturur** [reply]
yes i need to make a new release so the npm doc gets updated. The readme is already updated.
There is a lot of stuff to do everywhere, i can't continue working for v5.
Already putting up the old website and coordinating the domains has been an effort.

Also looking at an astro plugin for versioned docs for v7 onward.

**Reply by asturur** [reply]
Even just adding stuff to the website without breaking the layout is not easy

**Reply by aswind7** [reply]
> Even just adding stuff to the website without breaking the layout is not easy

Thank you for your hard work, and happy Christmas Eve!🎄🎅🏻🎅🏻

**Reply by JoyNop** [reply]
> [JoyNop](https://github.com/JoyNop) if you can mark this as answer you can help me. thanks.

Thank you very much for your help, this website is accessible normally, Merry Christmas🎅🏻
